QUIZ_PROGRAM-checkpoint.ipynb

```diff
@@ -0,0 +1,84 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ca5adba2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "327975a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('C:\\\\22BCA234\\\\DB&PYTHON\\\\emp.csv','w',newline='') as csvfile: \n",
```

Comment:
can i use only open

Reply:
Yes we can write... 
We are having two methods to open  a file:
                   (1) open() 
                   (2) with open() 
The minor difference between both the methods is that, that when we open our file with open() we need to close our file using close(). While when we open file with with open() the file is automatically closed... We don't need to close it..

Reply:
Done